lib/remap-functions/src/is_nullish.rs

```diff
@@ -13,7 +13,7 @@ impl Function for IsNullish {
     fn parameters(&self) -> &'static [Parameter] {
         &[Parameter {
             keyword: "value",
-            accepts: |v| matches!(v, Value::Bytes(_) | Value::Null),
+            accepts: |_| true,
```

Comment:
(Since there has been no issue description I just checked why this is correct): Looks fine because the nullish checks is defined for all types in this fallthrough: https://github.com/timberio/vector/blob/d31a2af3e0817eb488664ce262836b45f7c7d34c/lib/remap-functions/src/util.rs#L74

Reply:
Yeah, this was indeed a mismatch between how the function is meant to behave and which types the compiler expects. This is something we'll need to be vigilant about, as the function tests don't cover this.